src/Microsoft.DotNet.Interactive.Notebooks.Tests/DibNotebookDocumentFileFormatTests.cs

```diff
@@ -300,12 +295,13 @@ public void parsed_notebook_outputs_are_empty()
         public void notebook_can_be_parsed_from_different_extensions(string extension)
         {
             var fileName = $"notebook{extension}";
-            var notebook = ParseFromString(fileName, @"
+            var notebook = NotebookFileFormatHandler.Parse(fileName, @"
```

Comment:
These aren't strictly file formats, .dib in particular.

Reply:
Don't sure I get this, .dib is a serialization format, no?